packages/components/datetime/src/DateTime.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { CommonProps } from '@contentful/f36-core';
+
+import dayjs, { extend } from 'dayjs';
+import utcPlugin from 'dayjs/plugin/utc';
+extend(utcPlugin);
+
+export interface DateTimeProps extends CommonProps {
+  /**
+   * The date that will be displayed. It accepts a JS Date, an ISO8601 Timestamp string, or Unix Epoch Milliseconds number
+   */
+  date: Date | string | number;
+  /**
+   * The format in which the date will be presented
+   *
+   * @default 'full'
+   **/
+  format?: 'full' | 'time' | 'weekday' | 'day';
+  testId?: string;
+  className?: string;
+}
+
+function formatDateAndTime(
+  date: DateTimeProps['date'],
+  format: DateTimeProps['format'],
+): string {
+  let template: string;
+
+  switch (format) {
+    case 'day':
+      template = 'DD MMM YYYY'; // 17 Aug 2021
+      break;
+    case 'weekday':
+      template = 'ddd, DD MMM'; // Tue, 17 Aug
+      break;
+    case 'time':
+      template = 'h:mm A'; // 3:45 PM
+      break;
+    default:
+      template = 'ddd, DD MMM YYYY [at] h:mm A'; // Tue, 17 Aug 2021 at 3:45 PM
+  }
+
+  return dayjs(date).format(template);
+}
+
+function _DateTime(
+  { date, format = 'full', className, testId = 'f36-date-time' }: DateTimeProps,
```

Comment:
`testId` value doesn't follow `cf-ui` pattern

Reply:
we need to use {...otherProps} here

Reply:
yep yep!